src/network/networkd-json.c

```diff
@@ -62,6 +85,8 @@ static int address_build_json(Address *address, JsonVariant **ret) {
                                 JSON_BUILD_PAIR_FINITE_USEC("PreferredLifetimeUsec", address->lifetime_preferred_usec), /* for backward compat */
                                 JSON_BUILD_PAIR_FINITE_USEC("ValidLifetimeUSec", address->lifetime_valid_usec),
                                 JSON_BUILD_PAIR_FINITE_USEC("ValidLifetimeUsec", address->lifetime_valid_usec), /* for backward compat */
+                                JSON_BUILD_PAIR_FINITE_USEC("T1", lifetime_t1),
+                                JSON_BUILD_PAIR_FINITE_USEC("T2", lifetime_t2),
```

Comment:
T1 and T2 are not (only) for address but DHCP lease. We should add, say, `DHCPLease` field, instead.

Reply:
As "Address" field already has most of the DHCP lease options, won't it be appropriate to keep T1 and T2 in the Address field.  Please let me know if you think otherwise.
Sample output from JSON
{\"Family\":10,\"Address\":[32,1,0,0,0,0,0,0,0,0,0,0,0,0,0,16],\"PrefixLength\":128,\"Scope\":0,\"ScopeString\":\"global\",\"Flags\":512,\"FlagsString\":\"no-prefixroute\",**\"PreferredLifetimeUsec\":691202932072,\"ValidLifetimeUsec\":1036802932072,\"T1\":345600000000,\"T2\":552960000000**,\"ConfigSource\":\"DHCPv6\",\"ConfigState\":\"configured\",\"ConfigProvider\":[32,1,0,0,0,0,0,0,0,0,0,0,0,0,0,1]}]

Reply:
I mean, we should not add T1 and T2 in Address field. That's irrelevant for addresses, especially when it is assigned based on DHCP6 lease. The lifetimes are set based on IA_NA option.

Reply:
Ok understood. So if I understand the correct way would be to have DHCP6 field which will hold T1 and T2 values.